lib/phoenix/pubsub/pg2.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Phoenix.PubSub.PG2 do
+  use Supervisor
+
+  @moduledoc """
+  The Supervisor for the `:pg2` `Phoenix.PubSub` adapter
+
+  To use PG2 as your PubSub adapter, simply add it to your application's
+  supervision tree:
+
+      children = [
+        ...
+        worker(...),
+        supervisor(Phoenix.PubSub.PG2, [[name: MyApp.PubSub]]),
+      ]
+
+  ## Options
+
+    * `name` - The required name to register the PubSub processes, ie: `MyApp.PubSub`
+
+  """
+
+  def start_link(opts) do
```

Comment:
Let's make the name an argument (not an option) since it is now always required anyway? :)